routes/api.js

```diff
@@ -8,9 +8,9 @@
 
 'use strict';
 
-var expect = require('chai').expect;
-var MongoClient = require('mongodb');
-var ObjectId = require('mongodb').ObjectID;
+const expect = require('chai').expect;
+const MongoClient = require('mongodb');
```

Comment:
Are we not wanting to remove the inclusion of `mongodb` and all DB-related environment variables?

Reply:
@Sky020 Oh yes.  I forgot about that.  Actually, the following can be removed:

```js
const MongoClient = require('mongodb');
const ObjectId = require('mongodb').ObjectID;

const CONNECTION_STRING = process.env.DB; //MongoClient.connect(CONNECTION_STRING, function(err, db) {});
```
and the following files can be deleted:

- `tests/1_unit-tests.js`
- `.gitconfig`
- `.bashrc`
- `.profile`